grype/presenter/table/presenter.go

```diff
@@ -35,27 +43,26 @@ func (pres *Presenter) Present(output io.Writer) error {
 	rows := make([][]string, 0)
 
 	columns := []string{"Name", "Installed", "Fixed-In", "Type", "Vulnerability", "Severity"}
+	// Generate rows for matching vulnerabilities
 	for m := range pres.results.Enumerate() {
-		var severity string
+		row, err := createRow(m, false, pres.metadataProvider)
 
-		metadata, err := pres.metadataProvider.GetMetadata(m.Vulnerability.ID, m.Vulnerability.Namespace)
 		if err != nil {
-			return fmt.Errorf("unable to fetch vuln=%q metadata: %+v", m.Vulnerability.ID, err)
+			return err
 		}
+		rows = append(rows, row)
+	}
 
-		if metadata != nil {
-			severity = metadata.Severity
-		}
+	// Generate rows for suppressed vulnerabilities
+	if pres.inclSuppressed {
+		for _, m := range pres.ignoredMatches {
```

Comment:
... continuing from the previous thought of simplifying the presenter to just include the `ignoredMatches`, the `for _, m := range pres.ignoredMatches {` would just automatically skip if it's `nil`